packages/Bio/src/package.ts

```diff
@@ -423,17 +423,18 @@ export function multipleSequenceAlignmentAny(col: DG.Column<string> | null = nul
     const potentialCol = colInput.value;
     const unUsedName = table.columns.getUnusedName(`msa(${potentialCol.name})`);
   
-    if (checkInputColumnUi(potentialCol, 'MSA', [NOTATION.FASTA], ['DNA', 'RNA', 'PT'], false)) {
+    if (checkInputColumnUi(
+      potentialCol, potentialCol.name, [NOTATION.FASTA], [ALPHABET.DNA, ALPHABET.RNA, ALPHABET.PT], false)) {
       for (const inputRootStyle of inputRootStyles)
         inputRootStyle.display = 'none';
 
-      performAlignment = () => runKalign(potentialCol, false, unUsedName);
-    } else if (checkInputColumnUi(potentialCol, 'MSA', [NOTATION.HELM], [], false)) {
+      performAlignment = () => runKalign(potentialCol, false, unUsedName, clustersColInput.value);
+    } else if (checkInputColumnUi(potentialCol, potentialCol.name, [NOTATION.HELM], [], false)) {
       for (const inputRootStyle of inputRootStyles)
         inputRootStyle.display = 'initial';
 
-      performAlignment = () => runPepsea(
-        potentialCol, unUsedName, methodInput.value!, gapOpenInput.value!, gapExtendInput.value!);
+      performAlignment = () => runPepsea(potentialCol, unUsedName, methodInput.value!, gapOpenInput.value!,
```

Comment:
maybe unusedName, not unUsedName, don't think we need to make an accent on ...Used...